python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -574,7 +652,7 @@ def test_dag_fault_tolerance(ray_start_regular_shared):
     for i in range(99):
         ref = compiled_dag.execute(1)
         res = ray.get(ref)
-        if isinstance(res[0], RayDAGTaskError):
+        if isinstance(res[0], RayTaskError):
```

Comment:
We don't expect ray.get() to throw?

Reply:
Er sorry, what do you mean? We want ray.get() to throw an error?

Reply:
In other cases error is thrown: https://github.com/ray-project/ray/pull/46154/files#diff-95fd0ac9e0e48063233cc0b007e51fcad639e8d5cf5391c446deace83506e1ffR382
But why not here? I think I may be missing something.

Reply:
Oh I see, good catch! Let me look into that.